tokio-trace/src/macros.rs

```diff
@@ -1293,6 +1293,11 @@ macro_rules! callsite {
                 };
                 INTEREST.store(interest, Ordering::Relaxed);
             }
+
+            fn set_interest(&self, interest: Interest) {
+                INTEREST.store(interest, Ordering::Relaxed);
```

Comment:
I think this store needs to be sequentially consistent, to synchronize with the registry mutex (see https://github.com/tokio-rs/tokio/pull/1039#discussion_r273205330):
```suggestion
                INTEREST.store(interest, Ordering::SeqCst);
```

We should change `add_interest` and `clear_interest` as well, if those functions don't end up being removed.